### DISCUSSION TITLE
HistoryGraph min/max

### DISCUSSION THREAD

**Discussion by Bernd9000** [Zefau/ioBroker.jarvis#930] [Q&A]
Habe da mal eine Frage.
Ich möchte mir Luftdruckwerte anzeigen lassen.
Die Werte schwanken ja nur leicht zwischen 960 und 1060 mBar.
Das kann man im Diagramm natürlich kaum erkennen weil es hier eine gerade Linie ergibt.
Also habe ich bei "Konfiguration der y-Achse" in jarvis {"min":"960","max":"1060"} eingetragen.
Jetzt fehlen allerdings die horizontalen Linien im Diagramm und die Werte der y-Achse
Woran liegt das ? Gibt es auch ein automatischen Zoom ?

Mit min/max Angabe:
![1](https://user-images.githubusercontent.com/77005685/131750880-9160c94a-f308-44de-b357-9c2919096933.jpg)

Ohne min/max Angabe:
![2](https://user-images.githubusercontent.com/77005685/131751374-9affe4a8-efba-4cb1-97b4-47fc4513ab2a.jpg)

**Comment by Bernd9000** [comment]
Ok, Automatisch läuft es mit {"min":"dataMin","max":"dataMax"}
Bleibt noch die Frage warum bei manueller Angabe der Min/Max Werte die horizontalen Linien im Diagramm und die Werte der y-Achse fehlen.

**Comment by bmk666** [comment]
Hallo Bernd,

hast du doch noch das Problem mit den Chart? Also das die Werte nicht zu den Zeiten passen? Weil am Ende hast du wieder eine unregelmäßigkeit, so wie das ausseiht:

![image](https://user-images.githubusercontent.com/47809644/134011650-cc430061-3de1-49f6-bc56-89d6ee9c7966.png)

Grüße,
Sascha

**Reply by Bernd9000** [reply]
@bmk666 
Ja, Fehler ist noch vorhanden.
Werte passen nicht zu den Zeiten des History Adapter.
Und es werden Werte angezeigt die gar nicht vorhanden sind.
Ich vermute mal das hier eine "Glättung" der Werte durch Aggregation statt findet

**Comment by Bernd9000** [comment]
Um das nochmal zu verdeutlichen hab ich mit Node-red ein Taktgeber geschrieben der alle 10 Sekunden die Uhrzeit in einem Datenpunkt schreibt das dann durch den History Adapter aufgezeichnet wird. Damit ich auschliessen kann das da irgendwelche Umwandlungen im Datenfeld stattfinden hab ich die Uhrzeit als normale Ganzzahl kodiert.

Hier in dem Bild ist der erste Wert um 23:20 Uhr und 33 Sekunden aufgezeichnet worden (Wert : 232033)
Im chart steht als Uhrzeit 21:20 Uhr und 31 Sekunden.
Die chart Uhrzeiten liegen also zwei Stunden in der Vergangenheit.
Damit der Wert überhaupt angezeigt wird muß ich als "Rückblick" Wert im Widget mindestens 3 Stunden einstellen

![Unbenannt](https://user-images.githubusercontent.com/77005685/134249740-803b7891-0b8b-42fd-bb87-6f5c8a30e927.jpg)
![Unbenannt2](https://user-images.githubusercontent.com/77005685/134251263-0677c5b7-f20c-41df-860d-b693b7889764.jpg)
